src/dotnet/Chat.UI.Blazor/Components/LeftPanel/logo-button-svg.lit.ts

```diff
@@ -0,0 +1,47 @@
+import {customElement, property} from "lit/decorators.js";
+import {css, html, LitElement} from "lit";
+
+@customElement('logo-button-svg')
```

Comment:
I think it's useless to create web-components for just displaying SVG image. Just use img tag for that purpose.